app/assets/stylesheets/_settings.scss

```diff
@@ -324,15 +324,15 @@ $button-radius: 2px;
 // 11. Card
 // - - - - - - - - - - - - - - -
 
-// $card-background: #fff;
-// $card-color: isitlight($card-background);
-// $card-border: 1px solid smartscale($card-background, 7%);
-// $card-radius: $global-radius;
-// $card-shadow: 0 1px 2px rgba(#000, 0.2);
-// $card-padding: $global-padding;
-// $card-margin: 0.5rem;
+$card-background: #fff;
+$card-color: isitlight($card-background);
+$card-border: 1px solid ;
```

Comment:
https://github.com/sass/sass/issues/1088 is a big annoyance, so I'd suggest `.scss` (see discussion here: https://community.openproject.org/work_packages/18422) :stuck_out_tongue: